<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtherData</class>
 <widget class="QWidget" name="OtherData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>304</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;
border: solid #DFE5E9;
border-width: 1px 0px 0px 0px; </string>
  </property>
  <layout class="QVBoxLayout" name="OtherDataLayout">
   <property name="spacing">
    <number>9</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
color: #353739;</string>
     </property>
     <property name="text">
      <string>@eesti.ee post</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnCheckEMail">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>267</width>
       <height>0</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Roboto Condensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="accessibleName">
      <string extracomment="accessible">Check @eesti.ee e-mail status</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { padding: 6px 9px; border-radius: 2px; border: 1px solid #006EB5; color: #006EB5; background-color: #FFFFFF;}
QPushButton:pressed {  border: none; background-color: #006EB5; color: #FFFFFF;}
QPushButton:hover:!pressed { border: 1px solid #008DCF; color: #008DCF;}
QPushButton:disabled { border: 1px solid #BEDBED; color: #BEDBED;};
</string>
     </property>
     <property name="text">
      <string>CHECK @EESTI.EE E-MAIL STATUS</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblEMail">
     <property name="font">
      <font>
       <family>Roboto</family>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
color: #353739;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblNoForwarding">
     <property name="font">
      <font>
       <family>Roboto</family>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
color: #353739;</string>
     </property>
     <property name="text">
      <string>No official email redirects found. To send email to @eesti.ee, please enter your email address:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="inputEMail">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid #c8c8c8;
font-size: 14px;
color: #000000;
font-weight: 400;
text-decoration: underline;
</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="activate">
     <property name="minimumSize">
      <size>
       <width>267</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>267</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Roboto Condensed</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="accessibleName">
      <string extracomment="accessible">Activate redirection</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { padding: 6px 9px; border-radius: 2px; border: 1px solid #006EB5; color: #006EB5; background-color: #FFFFFF;}
QPushButton:pressed { border: 1px solid #41B6E6; color: #41B6E6;}
QPushButton:hover:!pressed { border: 1px solid #008DCF; color: #008DCF;}
QPushButton:disabled { border: 1px solid #BEDBED; color: #BEDBED;};
</string>
     </property>
     <property name="text">
      <string>ACTIVATE REDIRECTION</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>106</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="labelEestiEe">
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: none;
color: #75787B;</string>
     </property>
     <property name="text">
      <string>For more detailed official email address forwarding, please visit &lt;a href='http://www.eesti.ee/en/'&gt;&lt;span style='color: #006EB5; text-decoration: none; font-weight: 900;'&gt;eesti.ee&lt;/span&gt;&lt;/a&gt;. &lt;a href='https://www.eesti.ee/en/using-the-state-portal#terms-of-the-official-email-service-of-eestiee'&gt;&lt;span style='color: #006EB5; text-decoration: none; font-weight: 900;'&gt;Terms&lt;/span&gt;&lt;/a&gt; of the official email service of eesti.ee.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextBrowserInteraction</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
